pkg/cleaner/exec_process_cleaner.go

```diff
@@ -0,0 +1,14 @@
+package cleaner
```

Comment:
Could we also have some very basic unit testing coverage for this code? It can be pretty simple: Just use it to run `true` and `false`.

Reply:
Done!